tests/e2e/Jenkinsfile

```diff
@@ -24,7 +25,8 @@ pipeline {
         always {
           archiveArtifacts 'tests/e2e/results/*'
           junit 'tests/e2e/results/*.xml'
-          submitToActiveData('tests/e2e/results/py27.log')
+          submitToActiveData('tests/e2e/results/py27_raw.txt')
+          submitToTreeherder('go-bouncer', 'T', 'Tests', 'tests/e2e/results/py27_tbpl.txt')
```

Comment:
The arguments are project (repo), job symbol, job name, artifact path, log path. You're missing the artifact path, which should probably be 'tests/e2e/results/*'